daemon/api.go

```diff
@@ -985,7 +990,9 @@ func changeInterfaces(c *Command, r *http.Request, user *auth.UserState) Respons
 	}
 
 	change := state.NewChange(a.Action+"-snap", summary)
-	change.Set("snap-names", []string{a.Plugs[0].Snap, a.Slots[0].Snap})
+	change.Set("api-data", map[string]interface{}{
+		"snap-names": []string{a.Plugs[0].Snap, a.Slots[0].Snap},
```

Comment:
I added `api-data` here and with that I wrapped it, I don't particularly mind but it seems easier to read this way now that there is one more level in the data structure.